### ISSUE TITLE
Problem installing when User model is not `App\User`

### ISSUE THREAD

**Issue by wagnered** [cmgmyr/laravel-messenger#300]
Laravel 5.6 is installed.

The user providers array in auth.php is:
``` 
 'providers' => [
        'users' => [
            'driver' => 'eloquent',
            'model' => App\Models\User::class,
        ],`
```
Looks like 'messenger.user_model' is hard-coded as `App\User::class` and ignoring 'auth.providers.users.model'.

Regards...

**Comment by Gummibeer** [comment]
It's not hardcoded cause it has, like you wrote, a seperated config key:
https://github.com/cmgmyr/laravel-messenger/blob/master/config/config.php
`messenger.user_model`

**Comment by wagnered** [comment]
@Gummibeer 

Then please explain why, when I ran `composer update` the error I got was:
```
 Illuminate\Foundation\ComposerScripts::postAutoloadDump
 php artisan package:discover

In MessengerServiceProvider.php line 102:
                              
  Class 'App\User' not found  
```
It should automatically find `auth.providers.users.model'` and use that, not what was entered in config.php.

I could not run artisan at all until I changed `'user_model' => App\User::class` to `'user_model' => App\Models\User::class` in your config.php.

This means that the config.php file could not be published.

So, In my mind, it is ***effectively*** hard-coded, even though it wasn't intended to be.

I want to incorporate `cmgmyr/messenger` into the newest version of Ampache,  so please explain how I can get the package installed without modifying  `user_model` in the config.php file?

**Comment by a-komarev** [comment]
@wagnered thank you for reporting this issue! I've created PR #301 with fix.

/cc @cmgmyr

**Comment by a-komarev** [comment]
Its fixed, waiting for 2.16.1 release.

**Comment by wagnered** [comment]
Thank you,  much appreciated...
